tools/ci/tc/tests/test_valid.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 import jsone
-import mock
+from unittest import mock
```

Comment:
I don't like the order of this. @gsnedders @jgraham are there tools to fix/enforce import ordering in Python?